hq_superset/views.py

```diff
@@ -140,26 +149,58 @@ def array_is_falsy(array_values):
 
     return array_values
 
+# ~5MB
+ASYNC_DATASOURCE_IMPORT_LIMIT_IN_BYTES = 5000000
```

Comment:
Can this be moved to the top of the file or the superset config file, then we can read it from there?

Reply:
Sure. Let me move this to `utils.py` though, instead of the config file. Having it in a config file is required only if we need to customize it per env.